OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c

```diff
@@ -1018,6 +1059,31 @@ void getButcherTableau_RK1214(BUTCHER_TABLEAU* tableau) {
   tableau->isKRightAvailable = FALSE;
 }
 
+// Solving Stiff Systems of ODEs by Explicit Methods with Conformed Stability Domains
+// Anton E. Novikov1     Mikhail V. Rybkov2     Yury V. Shornikov3     Lyudmila V. Knaub4
+// EUROSIM 2016 & SIMS 2016
```

Comment:
Make this a Doxygen doc comment

```suggestion
/** @brief Get Runge-Kutta Ssc Butcher tableau.
 * Solving Stiff Systems of ODEs by Explicit Methods with Conformed Stability Domains
 * From:  Anton E. Novikov1     Mikhail V. Rybkov2     Yury V. Shornikov3     Lyudmila V. Knaub4
 * EUROSIM 2016 & SIMS 2016
 *
 * @param tableau    Pointer to Butcher tableau to fill.
 */
```